tests/test_integration.py

```diff
@@ -323,6 +323,67 @@ def test_lag_reg_deep(self):
             m.plot_parameters()
             plt.show()
 
+    def test_lag_regs(self):
```

Comment:
Please merge this test with the basic lagged regressor test - after merging the two methods into one.